src/app/docs/confirm/confirm-docs.component.html

```diff
@@ -0,0 +1,11 @@
+<sky-docs-demo-page
+  moduleName="SkyConfirmModule"
+  moduleSourceCodePath="src/app/public/modules/confirm/"
+  pageTitle="Confirm"
+>
+  <sky-docs-demo-page-summary>
+    The confirm service launches simple confirmation dialogs to allow users to confirm actions. The <sky-code>SkyConfirmService</sky-code> provider launches the confirmation dialog. Within the service, you can specify the dialog's message and customize the button text.
+
+  </sky-docs-demo-page-summary>
+
+</sky-docs-demo-page>
```

Comment:
Not sure where best to ask this, but @Blackbaud-SteveBrush, do we need to hide any of the properties that `docs-tools` exposes that were not previously included in the documentation?

Reply:
@blackbaud-johnly The `SkyModalScrollShadowEventArgs` interface needs to be hidden (`@internal`) but all other types look good to me.

Reply:
@Blackbaud-DeniseCisneros, do you have this? (FWIW, in the colorpicker repo, I merged the JSDocs PR without documenting the newly exposed properties as Steve suggested and [created an issue](https://github.com/blackbaud/skyux-colorpicker/issues/68) to circle back and document them.)

Reply:
Hidden. It was in the code but was just missing the `@` before internal :)